### PR TITLE
Update installation docs

### PR DESCRIPTION
**Pull request by daico007**
### PR Summary:
Remove `mosdef` channel from installation docs. 

### PR Checklist
------------
 - [x] Includes appropriate unit test(s)
 - [x] Appropriate docstring(s) are added/updated
 - [x] Code is (approximately) PEP8 compliant
 - [x] Issue(s) raised/addressed?

### CODE REVIEW

**Comment by mattwthompson** [comment]
Are you using anything from `omnia`? It comes up in search:

https://github.com/mosdef-hub/foyer/search?q=omnia

We are generally trying to move people away from it; now that the migration to `conda-forge` is nearly complete, it'll stop being maintained.

**Comment by daico007** [comment]
> Are you using anything from `omnia`? It comes up in search:
> 
> https://github.com/mosdef-hub/foyer/search?q=omnia
> 
> We are generally trying to move people away from it; now that the migration to `conda-forge` is nearly complete, it'll stop being maintained.

Good point! I will try remove `omnia` dependency for `foyer`.

File: docs/tmp/installation.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Install from conda:
 ```bash
-conda install -c omnia -c mosdef foyer
+conda install -c mosdef foyer
```
Comment (rsdefever) [inline docs/tmp/installation.md:3]: You still have the `mosdef` channel here. Why not `conda-forge`?

**Comment by rsdefever** [comment]
@daico007 excuse my confusion, the description of your PR says

> Remove `mosdef` channel from installation docs.

But your PR leaves in the `mosdef` channel in several places. It does however remove the `omnia` channel?

**Comment by daico007** [comment]
Closed since this PR is open from a my master branch. Will open a new one that addressed all the comments.

**Pull request closed without merging**